content/1.md

```diff
@@ -1,8 +1,103 @@
 ---
-title: 1 - Windows powertools & Firefox extensions
-description: Ensimmäinen episodi juhuu!
-date: 2021-01-01
+title: '1 – Windows powertools & Firefox extensions 🦾'
+description: |
+  Pyöräytetään uusi podi käyntiin metajaksolla!
+  Mitkä ovat devaajan parhaat tehotyökalut Windowsille?
+  Entä parhaat lisäosat Firefoxille?
+date: 2021-01-01 # TODO: Replace with real publication date
+recorded: 2021-03-19 # TODO: Is this correct?
 layout: episode
 ---
 
-CONTENTTIA
+## Microsoft PowerToys
+
+- 0:30 [Microsoft PowerToys][powertoys]
+- 0:42 FancyZones
+- 3:10 PowerToys Run
+- 6:52 Color Picker
+- 7:54 Image Resizer
+- 8:33 PowerRename
+- 9:15 Keyboard Manager
+- 9:35 File Explorer add-ons
+
+## Firefox-lisäosia
+
+- 10:14 Firefox-lisäosia
+- 10:23 [uBlock Origin][ublock-origin] (vs muut adblockerit)
+- 12:13 [Vimium][vimium]
+- 14:45 [Tree Style Tab][tree-style-tab]
+- 16:59 [LeechBlock NG][leechblock]
+- 19:03 [Refined GitHub][refined-github]
+- 20:18 [Monaco Markdown Editor For GitHub][monaco-markdown-github-editor]
+- (20:57 GitHubissa on nykyään dark mode,
+  jonka saa päälle [GitHubin asetuksista][github-settings])
+- 21:26 [Stylus][stylus]
+- 22:40 [Playback speed][playback-speed]
+  - Jaksossa kutsuttiin nimellä "Video speed controller"
+- 24:40 [Reddit Enhancement Suite (RES)][res]
+- 26:50 [Old Reddit Redirect][old-reddit-redirect]
+- 27:48 [Unhook &ndash; Remove YouTube Recommended Videos][unhook]
+- 28:33 [Imagus][imagus]
+
+## Muita softia
+
+- 29:26 [TouchCursor][touchcursor]
+- (31:39 [Ultimate Hacking Keyboard (UHK)][uhk])
+- 33:49 [KeyMouse][keymouse]
+- 36:10 [AutoHotkey (AHK)][ahk]
+  ja [AutoHotkey v2][ahk-v2]
+  sekä [Matiaksen koleat AutoHotkey-skriptit][ahk-scripts]
+- 41:10 [Lightshot][lightshot]
+- 41:52 [ShareX][sharex]
+  ja [Greenshot][greenshot]
+- 42:52 [Everything][everything]
+- 45:01 [Windows Task Scheduler][task-scheduler] (natiivi softa)
+- 45:14 [Hanneksen brutaali Auto shutdown -skripti][auto-shutdown]
+- 45:40 [Dorg: Hanneksen Downloads organizer -skripti][downloads-organizer]
+- 46:14 [Windows Clipboard][clipboard] (natiivi leikepöytämanageri)
+- 47:11 [TaskbarX][taskbarx]
+- 48:32 [f.lux][f.lux]
+  ja [Windows Night Light][night-light]
+- 49:27 [Windows Terminal][windows-terminal]
+
+## Jakson valinnat
+
+- 50:19 Jakson valinnat
+- 50:41 Matias: [Tailwind JIT][tailwind-jit]
+- 55:42 Hannes: [Androidin Display size -asetus][android-display-scaling]
+  - Jaksossa kutsuttiin nimellä "Display scaling"
```

Comment:
@Steellow Were you talking about the "Display size" setting? Is the link URL correct?